psychopy/scripts/psyexpCompile.py

```diff
@@ -182,7 +182,7 @@ def _makeTarget(thisExp, outfile, targetOutput):
             else:
                 # We already have the text
                 scriptText = scriptDict[scripts]
-            with io.open(eval(scripts), 'w', encoding='utf-8-sig') as f:
+            with io.open(eval(scripts), 'w', encoding='utf-8-sig', newline='\n') as f:
```

Comment:
Is this _really_ what we want? Shouldn't we take care of the line endings when _reading_ files?

Reply:
Good point, I will change from using codecs to read file to io, which has the "newline" parameter.